index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>My Portfolio</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  </head>
+
+  <body>
+    <header class="header">
+        <a href="#" class="logo">My Logo</a>
+        <nav id="nav-hamburger">
+            <label class="ham-icon" for="hamburger">&#9776;</label>            
+        </nav>
+    </header>
+    <section id="main">
+      <div id="background">
+        <img src="images/Rectangle 6.png" class="shape1">
+        <img src="images/Ellipse 11.png" class="shape2">
+        <img src="images/Ellipse 5.png" class="shape3">
+        <img src="images/Rectangle 7.png" class="shape4">
+        <img src="images/Ellipse 8.png" class="shape5">
+        <img src="images/Rectangle 4.png" class="shape6">
+        <img src="images/Ellipse 3.png" class="shape7">
+        <img src="images/Ellipse 12.png" class="shape8">
+        <img src="images/Ellipse 13.png" class="shape9">
+        <img src="images/Ellipse 15.png" class="shape10">
+        <img src="images/Ellipse 4.png" class="shape11">
+        <img src="images/Rectangle 5.png" class="shape12">
```

Comment:
- [ ] Great job 👏 adding the hamburger icon image from Figma, but you don't have to import all the icon images like this instead you can import the image as a whole:
   - ![firefox_eNHdetHkUb](https://user-images.githubusercontent.com/104968236/227176108-047137a4-5a1d-4ff2-ac60-3e39f1f3b0dd.png)
   - You can click on the "Normal Button/Tertiary/Enabled" link as shown above and go to the right sidebar and click on export.